Makefile

```diff
@@ -38,6 +38,7 @@ publish-release: $(GORELEASER)
 $(LINTER_VERSION_FILE):
 	rm -f $(LINTER)
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ./bin $(GOLANGCI_LINT_VERSION)
+	touch $(LINTER_VERSION_FILE)
```

Comment:
This fixes the `make lint` logic so it doesn't re-download the linter unnecessarily.